app/views/case-progress.html

```diff
@@ -39,7 +39,9 @@
     </tbody>
 </table>
 
+<div class="graph">  
```

Comment:
What's the graph div used for?

Reply:
Oh I forgot I added that in, will removed it now and push it up again!